addons/crm_project_issue/views/crm_lead_views.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <!-- inherit view of Lead Form : adding the "convert to issue" button -->
```

Comment:
@rpi-odoo  can you please tell me what is the need of this blank line ??
its nothing just increase LOC.